homeassistant/components/samsungtv/__init__.py

```diff
@@ -88,7 +92,7 @@ async def async_setup(hass, config):
 
 
 @callback
-def _async_get_device_bridge(data):
+def _async_get_device_bridge(data: dict[str, Any]) -> Any:
```

Comment:
```suggestion
def _async_get_device_bridge(data: dict[str, Any]) -> SamsungTVBridge:
```

Or a union of both bridge types

Reply:
Might be better with `SamsungTVLegacyBridge | SamsungTVWSBridge`

Reply:
Done